src/Features/ERDDAP/Platform/Observations/Table/table.tsx

```diff
@@ -1,14 +1,17 @@
 /**
  * Current observations table component
  */
-import * as React from "react"
+import React from "react"
 import { ListGroup, ListGroupItem } from "reactstrap"
 
+import { UnitSystem } from "Features/Units/types"
+
 import { RenderProps } from "../../Grabber"
 import { itemStyle, TableItem } from "./item"
 
-interface Props {
+interface Props extends RenderProps {
   unitSelector?: React.ReactNode
+  unit_system: UnitSystem
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)